test/support/api/teams.js

```diff
@@ -38,19 +40,19 @@ export function getTeam(data, { depth = 0 } = {}) {
 export function getTeams({ attributes, relationships, meta } = {}, { depth = 0 } = {}) {
   if (depth + 1 > 2) return [];
 
-  const clinicians = _.map(fxTestTeams, fxTeam => {
+  const teams = _.map(fxTestTeams, fxTeam => {
     const resource = getTeam(fxTeam, { depth });
 
     return mergeJsonApi(resource, { attributes, relationships, meta });
   });
 
-  return clinicians;
+  return teams;
 }
 
 // Exporting only teams needed for testing variance
-export const teamCoordinator = getTeamResource({ id: '11111' });
-export const teamNurse = getTeamResource({ id: '22222' });
-export const teamOther = getTeamResource({ id: '77777' });
+export const teamCoordinator = getTeamResource({ name: 'Coordinator' });
+export const teamNurse = getTeamResource({ name: 'Nurse' });
+export const teamOther = getTeamResource({ name: 'Other' });
```

Comment:
Like states, the `name` is not a unique value here. It would be better to use `abbr`, which is treated as a secondary unique ID.

Reply:
Ah okay good call, updated to that.